### PR TITLE
Metrics-WebApp

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Metrics Web Application

## Description
A mobile web application to check a list of metrics (numeric values) from an external API that provides numeric data about 
Covid-19 cases across different countries and continents.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,49 @@
-# metrics-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Metrics-Web App
+> A mobile web application to check a list of metrics (numeric values) from an external API that provides numeric data about Covid-19 cases across different countries and continents.
+
```
Comment (tufoinnkuo10) [inline README.md:5]: - [x] please ensure that the application is either deployed using Heroku, Netlify or GitHub Pages and A link to the deployed project is present in the README

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  justify-self: center;
+  margin: 0;
```
Comment (tufoinnkuo10) [inline src/App.css:5]: - [x] Great job with the design. there is this left to right scroll. please fix that.
- [ optional] to make the app more user friendly, you can ensure that when clicked on the continent name and not only the arrow icon it also takes you to the detail section 
demo2

File: src/redux/country.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    default:
+      return state;
+  }
+}
+
+// Action Creators
+export const getCountryInfo = (country) => async (dispatch) => {
+  const countryInfo = await getApiDataCountry(`${country}`);
+  // console.log(countryInfo);
```
Comment (tufoinnkuo10) [inline src/redux/country.js]: - [x] remove the console.log comment for clean code practices

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @ogagaoghene 

great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ design well implemented with home and details page
✅ test passing

## Required Changes ♻️
Check the comments under the review

- [ ]  please make another short brief video that contains a presentation of your work and a highlight about a piece of the code that you find interesting or you are proud of making.


tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (ogagaoghene) [reply README.md:5]: I have deployed the application to GitHub Pages and the deployed link is present in the README file

File: src/App.css (hunk 2 above)
Comment (ogagaoghene) [reply src/App.css:5]: I have addressed the horizontal scroll issue by adding max-width: 100% and overflow-x: hidden rule sets to the html, body selectors

File: src/redux/country.js (hunk 3 above)
Comment (ogagaoghene) [reply src/redux/country.js]: Thanks, I have removed the console.log comment as requested. It is indeed not a clean code practice

File: README.md [hunk 4]
```diff
@@ -1 +1,56 @@
...
+## Acknowledgments
+
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+
+- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) 
+
+## See Live
+
+[Live project to be added soon](https://ogagaoghene.github.io/metrics-app)
+
```
Comment (Bondok6) [inline README.md]: - [x] The Live demo link is broken, kindly fix it 👌

image

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @ogagaoghene 🙋‍♂️, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



## Suggested changes
- [ ] As the previous reviewer mentioned and according to the requirements you should highlight some interesting piece of code or something you built that you are very proud of in your video.

image

- [ ] _Check the comments under the review._

- ⭕ NOTE for Future Milestones: You have submitted a project, that has the wrong GitHub Flow. Your Pull Request should be from the `development` into the `main` branch. [for more info](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md). Please make sure to use the correct GitHub flow in future milestones  ⭕

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @ogagaoghene, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 4 above)
Comment (ogagaoghene) [reply README.md]: I have fixed this. I had to deploy it on netlify.

**Comment by ogagaoghene** [comment]
Thanks for approving my work.

**Comment by ogagaoghene** [comment]
Thank you

**Pull request merged**